.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "0.10"
```

Comment:
Is there another reason for dropping 0.10, other than it's old? As this'll mean a major version bump should we also drop 0.12?